### ISSUE TITLE
wrong classpath in crate.bat Crate 55.0

### ISSUE THREAD

**Issue by joalmjoalm** [crate/crate#3654]
classpath in crate.bat doesnt follow directory structure in the dist, ex sigar location.

crate 55.0
Windows 7 Pro, sp 1
Java client ver 8 update 91

log:
```
[2016-06-18 18:23:46,928][WARN ][bootstrap                ] jvm uses the client vm, make sure to run `java` with the server vm for best performance by adding `-server` to the command line
[2016-06-18 18:23:47,131][ERROR][bootstrap                ] Exception
java.nio.file.InvalidPathException: Illegal char <*> at index 143: C:\Users\d570116\Downloads\crate-0.55.0-201606180301-d06a02a.tar\crate-0.55.0-201606180301-d06a02a\crate-0.55.0-201606180301-d06a02a/lib/sigar/*
	at sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:182)
	at sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:153)
	at sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
	at sun.nio.fs.WindowsPath.parse(WindowsPath.java:94)
	at sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:255)
	at org.elasticsearch.common.io.PathUtils.get(PathUtils.java:60)
	at org.elasticsearch.bootstrap.JarHell.parseClassPath(JarHell.java:138)
	at org.elasticsearch.bootstrap.JarHell.parseClassPath(JarHell.java:96)
	at org.elasticsearch.bootstrap.JarHell.checkJarHell(JarHell.java:87)
	at org.elasticsearch.bootstrap.BootstrapProxy.setup(BootstrapProxy.java:171)
	at org.elasticsearch.bootstrap.BootstrapProxy.init(BootstrapProxy.java:296)
	at io.crate.bootstrap.CrateF.main(CrateF.java:50)
```

**Comment by chaudum** [comment]
hi @joalmjoalm thanks for reporting!
we'll take a look

**Comment by chaudum** [comment]
Hm, was automatically closed by Github.
The fix is in master and 0.55 branch and will be released with `0.55.1`. You can also test it with the next nightly build.

**Comment by chaudum** [comment]
@joalmjoalm `0.55.2 stable` was released yesterday
